helsingborg-stad/aterbruket-webapp
# create new verison after regive item. Changed some words to swedish

Explain the changes you've made
some words in the statistic page  and menu are translated to Swedish.
When a item is regiven there will be a new version of it in AWS.

Explain why these changes are made
There was missing a version and the status of the item in aws after regiving it. 

Explain your solution
changed the version counting and the status when regive the add. 

How to test the changes?
Go to Haffa statistik and see the translation.

Go AWS DynamoDB : 
- check so the status is changing after reserved, pickedUp, available.
- revisions should always be two numbers below the version. except the highest number that should be on version 0.
